indico/modules/events/registration/fields/base.py

```diff
@@ -44,6 +44,10 @@ class RegistrationFormFieldBase:
     not_required_validator = Optional
     #: the data fields that need to be versioned
     versioned_data_fields = frozenset({'price'})
+    #: the marshmallow base field for regform submission
+    mm_field_class = fields.Field
```

Comment:
if we always override this i'd set the default to `None` so it fails (not sure if a bare Field fails or just accepts anything as-is) if it's not being overridden

Reply:
From my testing a bare field accepts anything, so setting it to `None` is needed here